packages/api-explorer/src/components/DocMarkdown/DocComponents/Heading.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+import { Heading } from '@looker/components'
+import styled from 'styled-components'
+
+export const StyledHeading = styled(Heading)`
+  color: ${({ theme }) => theme.colors.text2};
+`
+
+StyledHeading.defaultProps = {
+  fontWeight: 'semiBold',
+  mb: 'xsmall',
+  pt: 'xsmall',
+}
```

Comment:
Curious why the headings don't use the stock theme color here? Perhaps that's something we should look more broadly if the value is wrong? 

On `mb` & `pt` – we're really advocating for Components to not have "built-in" margin (which it appears the padding might be acting as in a manner here) and instead to use Layout components like "Space" to create whitespace between components.

Reply:
@lukelooker - With every header the same color the hierarchy on the page suffers. I was trying to accomplish a visual hierarchy where H1 would be a bit darker than H2, for example. So in this case we passed it this color, but my understanding was that by using text2 we still maintain the themeable aspect which was the goal.

Reply:
**Closing remaining spacing comments per our conversation on Slack.**